operator/src/main/resources/scripts/startServer.sh

```diff
@@ -110,7 +110,19 @@ function waitForShutdownMarker() {
   # Wait forever.   Kubernetes will monitor this pod via liveness and readyness probes.
   #
   trace "Wait indefinitely so that the Kubernetes pod does not exit and try to restart"
+  logged=false
   while true; do
+    if [ "$logged" != 'true' ] && grep -q "BEA-141335" ${SERVER_OUT_FILE} ; then
+      msg=("WebLogic server failed to start due to missing or invalid"
+           "situational configuration files, which may be due to invalid"
+           "configOverride templates (these are specified via the"
+           "configOverride attribute in the Domain custom resource)."
+           "For details, please search your pod log or"
+           "${SERVER_OUT_FILE} for the keyword 'situational'."
+          )
+      trace "${msg[*]}"
```

Comment:
Two comments:
1) Looks like spaces are missing?  Do you need a space just before each trailing double-quote?
2) I keep going back and forth on this one - not sure if it's a concern - but I assume calling grep on the .out log every three seconds might actually incur significant overhead - maybe even enough to cause a big enough blip so as to affect latency sensitive apps. Maybe call this less often then 'doShutdown' is checked?

Reply:
For Tom's #2, what about running another script in the background like tailLog.sh that does the grep at periodic interval and that interval could even have an environment variable to change the sleep window?

I'm not suggesting this to make things more complicated but from trying to decipher the whole server lifecycle handling the process looks rather intricate so having a script to check the log means that same script can also be run or even exec'd independent of the lifecycle or other k8s probes.

Reply:
1. No need to add spaces in each line
2. Created monitorLog.sh script. Since we only monitor the fail to start server due to sit config error message, the script also monitor BEA-000360 (Server started) and exit when server has started. Sleep window by default is 30 seconds, and is configurable via environment variable.

Reply:
Nice! This looks much cleaner too...